run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+
+poetry install --only dev
```

Comment:
I actually get really confused if it's better to use `--with` or `--only` in this case.

Reply:
I decided to use `--only` so as not to interfere with how the TNA base layers poetry, which is moderately complicated